.github/CONTRIBUTING.md

```diff
@@ -40,9 +40,24 @@ A bug must have an issue tracking it in the issue tracker that has been approved
 
 ## Contributing features
 
-We're happy to discuss schema proposals as long as they align with our [Core Design Principles](http://adaptivecards.io/documentation/#about-overview). 
+Thank you for wanting to contribute to Adaptive Cards. If you are planning to make changes to the Core SDK, please make sure these changes align with our [Core Design Principles](http://adaptivecards.io/documentation/#about-overview). Prior to writing any code that affects rendering, please make sure you follow the following steps:
```

Comment:
The link to the design principals is using the wrong anchor... try this one instead

https://docs.microsoft.com/en-us/adaptive-cards/#core-design-principles